src/main/java/Ui.java

```diff
@@ -0,0 +1,129 @@
+                "_____________$$________________________________$$\n" +
+                "____________$$_________________________________$$\n" +
+                "____________$$_________________________________$$\n" +
+                "____________$$___________________________________$\n" +
+                "____________$$___________________________________$$\n" +
+                "__________$$_________________________$$___________$\n" +
+                "__________$$__________$$___________$$_____________$$\n" +
+                "________$$__$$________$$_________$$_______________$$\n" +
+                "______$$____$$__________$$_______$$_______________$$\n" +
+                "______$$____$$____________$$___$$_________________$$\n" +
+                "____$$______$$_____________$$_$$_______$$_________$$\n" +
+                "____$$______$$________$$____$$$________$$_________$$\n" +
+                "____$$______$$________$$____$$$_______$$__________$$\n" +
+                "____$$______$$________$$_______________$$__________$$\n" +
+                "____$$______$$________$$_______________$$____________$\n" +
+                "_$$$$_______$$________$$_______________$$____________$$\n" +
+                "$___$$______$$________$$$$___________$$$$____________$$\n" +
+                "$___$$______$$________$$__$$_______$$__$$____________$$\n" +
+                "_$$$$$______$$________$$____$$___$$_____$$___________$$\n" +
+                "____$$______$$________$$______$$_______$$___________$$\n" +
+                "____$$______$$________$$_____$$________$$___________$$\n" +
+                "__$$________$$________$$$$$$$$___$$$$$$__$$_________$$\n" +
+                "__$$________$$________$$______$$$______$$$$_________$$\n" +
+                "$$________$$__________$$_________$$$$$$__$$__________$\n" +
+                "$$______$$__________$$$$$$$$$$$$$$$______$$__________$\n" +
+                "$$_$$_$$$__________$$_____________$$$$$$$__$$_________$\n" +
+                "_$$$$$$$___________$$______________________$$________$$\n" +
+                "_____$$__$$__$$__$$_$______________________$$__________$$\n" +
+                "______$$$$__$___$__$$______________________$$____________$\n" +
+                "_______$$___$___$__$________________________$$_$__$$__$$__$\n" +
+                "_________$$$$$$$$$$__________________________$$_$_$$$$$$$$\n");
+        System.out.println("What can this good boi do for you?");
+        System.out.println("--------------------------------------");
+    }
+
+    /**
+     * Prints a given string but adds "Added : " in front of the message.
+     * @param msg given string
+     */
+    public void taskPrint(String msg) {
+        System.out.println("Added : " + msg);
+    }
+
+
+    /** Prints out a given ArrayList of tasks in a nice format, with the
+     * given displayMessage on top.
+     * @param toDoList given todolist
+     * @param displayMessage message to be displayed
```

Comment:
If I am not wrong, I think the first letter of each sentence of the JavaDoc comments should be capitalized? I noticed a similar issue in other JavaDoc comments as well. Perhaps you could capitalize "given" and "message" in this case?